Dockerfile

```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1
```

Comment:
This file doesn't seem to have much purpose, perhaps it can be removed? BuildMaster has its own docker setup.

Reply:
At this moment I agree that this file can be removed. Maybe in the future it would be helpful for developing HaikuPorter, i.e. some integration tests.